src/rooms/list/v1.rs

```diff
@@ -0,0 +1,191 @@
+//! [GET /_synapse/admin/v1/rooms](https://github.com/matrix-org/synapse/blob/master/docs/admin_api/rooms.md#list-room-api)
+use js_int::UInt;
```

Comment:
Just remembered that we re-export this from `ruma`. For the next PR, you can just import it from there.